ci-operator/config/openshift/etcd/openshift-etcd-openshift-4.16__okd-scos.yaml

```diff
@@ -16,23 +16,14 @@ images:
 - build_args:
   - name: TAGS
     value: scos
-  dockerfile_path: Dockerfile.rhel
+  dockerfile_path: Dockerfile.art
   from: origin_scos-4.16_base-stream9
   inputs:
     ocp_builder_rhel-9-etcd-golang-1.19:
       as:
+      - builder
       - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.16
   to: etcd
-- build_args:
-  - name: TAGS
-    value: scos
-  dockerfile_path: Dockerfile.installer
-  from: origin_scos-4.16_base-stream9
-  inputs:
-    ocp_builder_rhel-9-etcd-golang-1.19:
-      as:
-      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.16
-  to: installer-etcd-artifacts
```

Comment:
installer-etcd-artifacts image needs to be added back (to keep in accordance with other etd configs)